app/src/features/rules/components/RulesList/components/RulesTable/utils/rules.ts

```diff
@@ -10,7 +10,26 @@ export const convertToArray = <T>(record: T | T[]): T[] => {
 };
 
 export const getActiveRules = (records: RuleObj[]) => {
-  return records.filter((record) => record.status === RuleObjStatus.ACTIVE);
+  let activeRecords: RuleObj[] = [];
+  const seenGroupIds = new Set<string>();
+  const rules: RuleObj[] = [];
+  const groups: Record<string, RuleObj> = {};
+
+  records.forEach((record) => {
+    if (isRule(record) && record.status === RuleObjStatus.ACTIVE) {
+      rules.push(record);
+    } else {
+      groups[record.id] = record;
+    }
```

Comment:
this will add inactive rules inside the groups object. 
not a problem now but will become a source of bugs later